extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -116,6 +116,10 @@ amp-carousel .amp-carousel-button.amp-disabled {
   width: 100% !important;
 }
 
+.-amp-slide-item > * {
+  flex: 0 0 100% !important;
```

Comment:
So, I'm thinking, since the old version had height 100% as well, let's do it here too? `height: 100%`?